test/integration/admin_crud_specialties_test.rb

```diff
@@ -5,7 +5,9 @@ class AdminCrudSpecialtiesTest < ActionDispatch::IntegrationTest
     admin = create(:admin)
     ApplicationController.any_instance.stubs(:current_admin).returns(admin)
 
-    visit new_admin_specialty_path
+    visit 'admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.